rsnapshot.conf Debian configuration choices:
- Change (default commented) log filename as per Debian Policy section 10.8
- Change default Debian directory for snapshots
- Change path for rsnapshot-diff
- Uncomment rsync link_dest support
- Manpage: point to the real default file on Debian systems
- Replace interval (deprecated alias) with retain

--- a/rsnapshot.conf.default.in
+++ b/rsnapshot.conf.default.in
@@ -24,7 +24,7 @@
 
 # All snapshots will be stored under this root directory.
 #
-snapshot_root	/.snapshots/
+snapshot_root	/var/cache/rsnapshot/
 
 # If no_create_root is enabled, rsnapshot will not automatically create the
 # snapshot_root directory. This is particularly useful if you are backing
@@ -68,7 +68,7 @@
 
 # Uncomment this to specify the path to rsnapshot-diff.
 #
-#cmd_rsnapshot_diff	/usr/local/bin/rsnapshot-diff
+#cmd_rsnapshot_diff	/usr/bin/rsnapshot-diff
 
 # Specify the path to a script (and any optional arguments) to run right
 # before rsnapshot syncs files
@@ -86,10 +86,10 @@
 # i.e. hourly, daily, weekly, etc.      #
 #########################################
 
-interval	hourly	6
-interval	daily	7
-interval	weekly	4
-#interval	monthly	3
+retain		hourly	6
+retain		daily	7
+retain		weekly	4
+#retain		monthly	3
 
 ############################################
 #              GLOBAL OPTIONS              #
@@ -113,7 +113,7 @@
 # If you enable this, data will be written to the file you specify. The
 # amount of data written is controlled by the "loglevel" parameter.
 #
-#logfile	/var/log/rsnapshot
+#logfile	/var/log/rsnapshot.log
 
 # If enabled, rsnapshot will write a lockfile to prevent two instances
 # from running simultaneously (and messing up the snapshot_root).
--- a/rsnapshot-program.pl
+++ b/rsnapshot-program.pl
@@ -6120,10 +6120,10 @@
 =head1 CONFIGURATION
 
 B</etc/rsnapshot.conf> is the default configuration file. All parameters
-in this file must be separated by tabs. B</etc/rsnapshot.conf.default>
+in this file must be separated by tabs. B</usr/share/doc/rsnapshot/examples/rsnapshot.conf.default.gz>
 can be used as a reference.
 
-It is recommended that you copy B</etc/rsnapshot.conf.default> to
+It is recommended that you copy B</usr/share/doc/rsnapshot/examples/rsnapshot.conf.default.gz> to
 B</etc/rsnapshot.conf>, and then modify B</etc/rsnapshot.conf> to suit
 your needs.
 
@@ -6917,7 +6917,7 @@
 =head1 NOTES
 
 Make sure your /etc/rsnapshot.conf file has all elements separated by tabs.
-See /etc/rsnapshot.conf.default for a working example file.
+See /usr/share/doc/rsnapshot/examples/rsnapshot.conf.default.gz for a working example file.
 
 Make sure you put a trailing slash on the end of all directory references.
 If you don't, you may have extra directories created in your snapshots.
